### PR TITLE
Add a flag for Istio version, also run tests using 'stable' version.

### PR DESCRIPTION
**Pull request by tcnghia**

### CODE REVIEW

File: third_party/istio-stable [hunk 1]
```diff
@@ -0,0 +1 @@
+istio-1.4.6
```
Comment (mattmoor) [inline third_party/istio-stable]: Add trailing newline:
```suggestion
istio-1.4.6

```

File: third_party/istio-stable [hunk 2]
```diff
@@ -0,0 +1 @@
+istio-1.4.6/
```
Comment (mattmoor) [inline third_party/istio-stable]: Add trailing newline:
```suggestion
istio-1.4.6/

```

Comment (mattmoor) [inline third_party/istio-stable]: Add trailing newline:
```suggestion
istio-1.4.6/

```

**Comment by tcnghia** [comment]
@cla-bot I signed it
@google-bot I signed it

**Comment by tcnghia** [comment]
@google-bot I signed it

File: third_party/istio-stable (hunk 2 above)
Comment (mattmoor) [inline third_party/istio-stable]: Add trailing newline:
```suggestion
istio-1.4.6/

```

**Pull request merged**